esp/services/WsDeploy/WsDeployService.cpp

```diff
@@ -5338,7 +5338,10 @@ bool CWsDeployFileInfo::handleRows(IEspContext &context, IEspHandleRowsRequest &
       }
 
       if (pComponent)
+      {
+        xpath.append("[1]");
```

Comment:
are we expecting this path to be otherwise ambiguous? Is there a particular reason why we are fetching the first element, and it's safe to ignore the other elements??